delta/ml/predict.py

```diff
@@ -471,7 +471,7 @@ def _process_block(self, pred_image, y, x, labels, label_nodata):
 
             if self._metrics:
                 for m in self._metrics:
-                    m.update_state(valid_labels.compressed(), valid_pred.compressed()) #pylint: disable=E1101
```

Comment:
I think you need the compressed to not count masked values... Is there a reason it was removed?

Reply:
The compressed call converts the data to a 1D array, making it incompatible with the msssim metrics.  I don't know if there is a good way to use msssim with masked values.

Reply:
Oh, good point, I hadn't thought of that! MSSSIM doesn't make much sense to use here... the more common case would be something like precision / recall, where we would definitely want to filter out the nodata pixels. I think it might be better to state we only support metrics on individual pixels here. What do you think @mvonpohle ?